lib/ansible/plugins/callback/default.py

```diff
@@ -188,7 +188,7 @@ def _task_start(self, task, prefix=None):
 
         # Preserve task name, as all vars may not be available for templating
         # when we need it later
-        if self._play.strategy == 'free':
+        if self._play.strategy.get('lockstep', False) is False:
```

Comment:
I would assume True as that is the default for most strategies

Reply:
Yeah, I was debating this, and I picked False.  I'll think it over more.